Orm/Xtensive.Orm/Xtensive.Orm.csproj

```diff
@@ -49,7 +49,7 @@
   <ItemGroup Condition="'$(TargetFramework)'=='net5.0'">
     <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)'=='net6.0'">
+  <ItemGroup Condition="'$(TargetFramework)'!='net5.0'">
```

Comment:
This has nothing to do with visitors. I also have a PR which removes net5 from supported frameworks and adds net7, which makes this change kind of unnecessary.

Reply:
Reverted, but equality condition is time-bomb anyway.

The project should be compilable with any future .NET SDK version without modification.

Reply:
I will add default condition for frameworks which are not NET6 and not NET7 with package for NET7, but this doesn't mean the project will be compatible with ANY FUTURE .NET . It is impossible. 😉  For example, code that was compliable in NET6 happened to have breaking changes in NET7, I even see it here, you had to fix it. So our guarantee of project being successfully built ends with supported target frameworks, any newer frameworks can bring any other breaking change and we don't have a crystal ball.

Reply:
Of course, there may be many unknown factors in the future
The best we can: don't limit compatibility artifically